lib/ship_quiet_logistics/commands/processor.rb

```diff
@@ -11,20 +11,28 @@ def initialize(blackboard, queue)
 
         @config = ShipQuietLogistics.configuration
       end
-
       def call
         messages_count.times do
           document = next_document
           next if !document || document.nil?
+          case document_type document.namespace
+            when 'RMADocument'
+              config.process_rma_handler.(document)
```

Comment:
@hugobast I added the config and the class, however it is not yet used in the processor.
This will probably be used when I do the other part of this integration, but it may be out of scope of this ticket.
Would you rather have me delete it all ?